vllm/attention/backends/rocm_flash_attn.py

```diff
@@ -299,7 +300,8 @@ def __init__(
         else:
             # if not using triton, navi3x/navi21/navi10 do not use flash-attn
             # either
-            if torch.cuda.get_device_capability()[0] != 9:
+            current_capability = current_platform.get_device_capability()
```

Comment:
you can embed this logic in `has_device_capability` ?

Reply:
This involves an exact match which is different from `has_device_capability` which only requires a minimum version. Maybe I can come up with a new function for this...

Reply:
`torch.cuda.get_device_capability()[0] != 9` is essentially `torch.cuda.get_device_capability()[0] < 9` , because 9 is the largest one.

Reply:
I see, not sure why it was written that way originally then... I'll update the code as you have said.

Reply:
I don't see any problem